packages/block-library/src/block/edit.js

```diff
@@ -444,7 +230,7 @@ function ReusableBlockEdit( {
 
 	const resetContent = () => {
 		if ( content ) {
-			replaceInnerBlocks( patternClientId, initialBlocks );
+			setAttributes( { content: undefined } );
```

Comment:
What is the content attribute?

Reply:
In the pattern block, when you override a block attribute, it adds a new "content" attribute pointing to the name of the block. Something like this:

`<!-- wp:block {"ref":1602,"content":{"Custom Name":{"content":"Edited paragraph content"}}} /-->`

Reply:
Also, do we declare this attribute in the block type registration?

Reply:
Ah, I see this is a preexisting attribute

Reply:
> I wonder if there's a better attribute name we can use. Maybe `overrides`?

It's used to be called `overrides`, but the feedback was to change it to `content`.